scripts/OcsBasicScript.txt

```diff
@@ -0,0 +1,27 @@
+Rem --- eWON start section: Cyclic Section

```

Comment:
Why is this not .bas extension?

Reply:
ah, that's what it was. It seems to not even look at the extension but that's good to know, will change